lib/active_encode/base.rb

```diff
@@ -13,4 +13,6 @@ class Base
     include EngineAdapter
     include Callbacks
   end
+
+  class NotFound < RuntimeError; end
```

Comment:
What would you think about either subclassing `ArgumentError`, or directly using `ArgumentError` with a custom message, since this is a condition that arises when passing an unexpected argument (`nil`) to the `#find` method?